### PR TITLE
feat: Create tests.yml

### PR DESCRIPTION
**Pull request by noamisraeli**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        # Run in all these versions of Python
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9]
```
Comment (Avivsalem) [inline .github/workflows/tests.yml:23]: test 3.7 and up.
no need for 3.5, 3.6

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      - name: Set Up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+        # Display the Python version being used
+      - name: Display Python version
+        run: python -c "import sys; print(sys.version)"
+        # Install the package using the setup.py
+      - name: Install package
+        run: python setup.py install
```
Comment (Avivsalem) [inline .github/workflows/tests.yml:39]: why not pip install.

**Review by Avivsalem — CHANGES_REQUESTED** [review]

File: .github/workflows/tests.yml (hunk 1 above)
Comment (noamisraeli) [reply .github/workflows/tests.yml:23]:
```suggestion
        python-version: [3.7, 3.8, 3.9]
```

File: .github/workflows/tests.yml (hunk 2 above)
Comment (noamisraeli) [reply .github/workflows/tests.yml:39]:
```suggestion
        run: python install .
```

**Review by Avivsalem — APPROVED** [review]

**Pull request merged**